packages/react/src/components/library/task-list-grid/TaskListGrid.tsx

```diff
@@ -2,13 +2,11 @@ import React, { useEffect, useState, useCallback } from 'react';
 import { useNavigate } from 'react-router-dom';
 
 import DataGrid, {
-  Column, Selection, Sorting, HeaderFilter,
+  Column, Selection, Sorting, HeaderFilter, DataGridTypes,
```

Comment:
Minor: most of these types are accessible through DataGridTypes as well, their imports can be removed